src/MessagePack/BufferWriter.cs

```diff
@@ -139,6 +139,19 @@ public void Write(ReadOnlySpan<byte> source)
             }
         }
 
+        /// <summary>
+        /// Writes a sequence to the writer.
+        /// </summary>
+        /// <param name="source">The sequence to be copied.</param>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal void Write(ReadOnlySequence<byte> source)
```

Comment:
Moves the code formerly in `Utilties.cs` here. `Utilities` references Nerdbank.Streams and the type is not external, so making this an instance method seems correct.

Reply:
This wasn't in `BufferWriter` because `BufferWriter` came from the .NET team and I was trying to keep the two files relatively in-sync. So I'm mixed on this change.

Reply:
Could you point me to where the type exists? I couldn't find a BufferWriter in https://github.com/dotnet/corefx

Reply:
It's here: https://github.com/aspnet/AspNetCore/blob/master/src/Servers/Kestrel/Core/src/Internal/BufferWriter.cs

But I see I've already diverged significantly from that version, so I guess this change is fine.